app/controllers/mentee_profiles_controller.rb

```diff
@@ -1,6 +1,24 @@
 class MenteeProfilesController < ApplicationController
-	def show
-	end
-	def dashboard
-	end
+  def show
+    @user = User.find_by(id: params[:id])
```

Comment:
in show , i have searched user by id, becuase organiser who are login can also see the show page, so shouldnt search by current_user.  Is it right ?